src/KeyAwareInterface.php

```diff
@@ -16,7 +18,7 @@
      *
      * @since 0.1
      *
-     * @return string The key that this interface has.
+     * @return string|Stringable|null The key that this interface has.
```

Comment:
I don't like how this description references the "interface". Consider revising.

Reply:
Suggestion:

> The key associated with this instance.